docs/translations.json

```diff
@@ -1,126 +1,126 @@
+  "Data store reference",
+  "Default flow",
   "Distribute elements horizontally",
   "Distribute elements vertically",
-  "Divide into three Lanes",
-  "Divide into two Lanes",
-  "Empty Pool",
-  "Empty Pool (removes content)",
-  "End Event",
-  "Error Boundary Event",
-  "Error End Event",
-  "Error Start Event",
-  "Escalation Boundary Event",
-  "Escalation Boundary Event (non-interrupting)",
-  "Escalation End Event",
-  "Escalation Intermediate Throw Event",
-  "Escalation Start Event",
-  "Escalation Start Event (non-interrupting)",
-  "Event Sub Process",
-  "Event based Gateway",
-  "Exclusive Gateway",
-  "Inclusive Gateway",
-  "Intermediate Throw Event",
-  "Link Intermediate Catch Event",
-  "Link Intermediate Throw Event",
+  "Divide into three lanes",
+  "Divide into two lanes",
+  "Empty pool/participant",
+  "Empty pool/participant (removes content)",
+  "End event",
+  "Error boundary event",
+  "Error end event",
+  "Error start event",
+  "Escalation boundary event",
+  "Escalation boundary event (non-interrupting)",
+  "Escalation end event",
+  "Escalation intermediate throw event",
+  "Escalation start event",
+  "Escalation start event (non-interrupting)",
+  "Event sub-process",
+  "Event-based gateway",
+  "Exclusive gateway",
+  "Inclusive gateway",
+  "Intermediate throw event",
+  "Link intermediate catch event",
+  "Link intermediate throw event",
   "Loop",
-  "Manual Task",
-  "Message Boundary Event",
-  "Message Boundary Event (non-interrupting)",
-  "Message End Event",
-  "Message Intermediate Catch Event",
-  "Message Intermediate Throw Event",
-  "Message Start Event",
-  "Message Start Event (non-interrupting)",
-  "Parallel Gateway",
-  "Parallel Multi Instance",
-  "Participant Multiplicity",
-  "Receive Task",
+  "Manual task",
+  "Message boundary event",
+  "Message boundary event (non-interrupting)",
+  "Message end event",
+  "Message intermediate catch event",
+  "Message intermediate throw event",
+  "Message start event",
+  "Message start event (non-interrupting)",
+  "Parallel gateway",
+  "Parallel multi-instance",
+  "Participant multiplicity",
+  "Receive task",
   "Remove",
-  "Script Task",
-  "Send Task",
-  "Sequence Flow",
-  "Sequential Multi Instance",
-  "Service Task",
-  "Signal Boundary Event",
-  "Signal Boundary Event (non-interrupting)",
-  "Signal End Event",
-  "Signal Intermediate Catch Event",
-  "Signal Intermediate Throw Event",
-  "Signal Start Event",
-  "Signal Start Event (non-interrupting)",
-  "Start Event",
-  "Sub Process",
-  "Sub Process (collapsed)",
-  "Sub Process (expanded)",
+  "Script task",
+  "Send task",
+  "Sequence flow",
+  "Sequential multi-instance",
+  "Service task",
+  "Signal boundary event",
+  "Signal boundary event (non-interrupting)",
+  "Signal end event",
+  "Signal intermediate catch event",
+  "Signal intermediate throw event",
+  "Signal start event",
+  "Signal start event (non-interrupting)",
+  "Start event",
+  "Sub-process",
```

Comment:
In the docs, it's treated as a single word: https://docs.camunda.io/docs/components/modeler/bpmn/subprocesses/
While in the specification, it takes form of "Sub-Process". I think the hyphened version is correct here.